### PR TITLE
Change to SpatiaLite

### PR DESCRIPTION
**Pull request by havatv**
Goal: Make sure that SpatiaLite is used throughout the documentation (and some updates to the spatialite document - numbered lists, line length adjustments).
Trivial updates - no need for review.

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/training_manual/databases/spatialite.rst [hunk 1]
```diff
@@ -1,70 +1,77 @@
...
+can do at this point is to delete the connection.
+This is of course because we haven't added any tables to this database.
+Let's go ahead and do that.
+
+#. Find the button to create a new layer and use the dropdown to
+   create a new new SpatiaLite layer, or select
+   :menuselection:`Layer --> New --> New SpatiaLite Layer`.
+
+   |newSpatiaLiteLayer|
+
```
Comment (DelazJ) [inline docs/training_manual/databases/spatialite.rst:42]:
```suggestion
   :menuselection:`Layer --> New -->` |newSpatiaLiteLayer|
   :menuselection:`New SpatiaLite Layer`.
```

File: docs/training_manual/databases/spatialite.rst [hunk 2]
```diff
@@ -1,70 +1,77 @@
...
+#. Select the database we created in the previous steps in the drop
+   down.
+#. Give the layer the name :kbd:`places`.
+#. Tick the checkbox next to
+   :guilabel:`Create an auto-incrementing primary key`.
+#. Add 2 attributes as shown in below
```
Comment (DelazJ) [inline docs/training_manual/databases/spatialite.rst:48]: 2 vs two?

File: docs/training_manual/databases/spatialite.rst [hunk 3]
```diff
@@ -1,70 +1,77 @@
...
+your new SpatiaLite DB.
 
 |IC|
--------------------------------------------------------------------------------
+----------------------------------------------------------------------
 
-You have seen how to create spatialite databases and add tables to them and to
-use these tables as layers in QGIS. 
+You have seen how to create SpatiaLite databases and add tables to
+them and to use these tables as layers in QGIS.
```
Comment (DelazJ) [inline docs/training_manual/databases/spatialite.rst:74]:
```suggestion
You have seen how to create SpatiaLite databases, add tables to
them and use these tables as layers in QGIS.
```

File: docs/training_manual/databases/spatialite.rst [hunk 4]
```diff
@@ -1,70 +1,77 @@
...
+
+#. Find the button to create a new layer and use the dropdown to
+   create a new new SpatiaLite layer, or select
+   :menuselection:`Layer --> New --> New SpatiaLite Layer`.
+
+   |newSpatiaLiteLayer|
+
+#. Select the database we created in the previous steps in the drop
+   down.
+#. Give the layer the name :kbd:`places`.
```
Comment (DelazJ) [inline docs/training_manual/databases/spatialite.rst:45]:
```suggestion
#. Give the layer the name ``places``.
```

File: docs/training_manual/databases/spatialite.rst [hunk 5]
```diff
@@ -1,70 +1,77 @@
...
+#. Tick the checkbox next to
+   :guilabel:`Create an auto-incrementing primary key`.
+#. Add 2 attributes as shown in below
+#. Click :guilabel:`OK` to create the table.
+
+   .. figure:: img/new_layer_setup.png
+      :align: center
+
+#. Click the refresh button at the top of the Browser and you should
+   now see your :kbd:`places` table listed.
```
Comment (DelazJ) [inline docs/training_manual/databases/spatialite.rst:55]:
```suggestion
   now see your ``places`` table listed.
```

**Comment by havatv** [comment]
> > no need for review
> 
> Hum hum...

What I meant was that there was no need to review the changes (and as far as I can see, you have no suggestions that affect the changes that were made in the PR). That there are room for further improvements is another matter. I did not notice the kbd misuse.

File: docs/training_manual/databases/spatialite.rst (hunk 2 above)
Comment (havatv) [inline docs/training_manual/databases/spatialite.rst:48]:
```suggestion
#. Add two attributes as shown in below
```

Comment (havatv) [reply docs/training_manual/databases/spatialite.rst:48]: definitely "two".

File: docs/training_manual/databases/db_manager.rst [hunk 6]
```diff
@@ -203,13 +203,13 @@ command line tools, so now let's learn how to use DB Manager to do imports.
 .. figure:: img/import_layer_btn.png
    :align: center
 
-* Select the :kbd:`urban_33S.shp` file from :kbd:`exercise_data/projected_data`
-  as the input dataset.
+* Select the file:`urban_33S.shp` file from
```
Comment (havatv) [inline docs/training_manual/databases/db_manager.rst:206]:
```suggestion
* Select the :file:`urban_33S.shp` file from
```

**Pull request merged**